userpanel/modules/documents/style/bclean/templates/documents.html

```diff
@@ -44,8 +42,8 @@ <h4 class="modal-title">{trans("Authorization")}</h4>
 				</div>
 			</div>
 			<div class="modal-footer">
-				<button type="button" class="btn btn-danger submit-button" formnovalidate accesskey="S">{trans("<!helpdesk>Submit")}</button>
-				<button type="button" class="btn btn-secondary" data-dismiss="modal">{trans("Close")}</button>
+				{button type="submit" class="btn btn-danger submit-button" accesskey="S" icon="submit" label="<!helpdesk>Submit" formnovalidate=true}
```

Comment:
Dlaczego tu zmienasz typ buttona z `button` na `submit`? Zdajesz sobie sprawę, że w efekcie tego kliknęcie guzika będzie powodowało natychmiastowe przesłanie formularza do serwera www, a to może być niepożądane i np. prowadzić do dwukrotnego pzesłania.

Reply:
byłem pewien że właśnie ten typ chroni przed tym - cofnąłem zmianę typu https://github.com/lmsgit/lms/pull/1997/commits/b0a0dc356ffbf30f23efd2e728cdea68a4559b32